### PR TITLE
Fix wording in charter page

### PR DESCRIPTION
**Pull request by vannyle**
**Describe what changes this pull request brings**
This pull request brings the Charter page.

**Additional changes**
- Prevented the horizontal scroll by changing the absolute positioning of the submenu in the header navigation.

**Steps to steps**
1. Open [the page](https://deploy-preview-171--ebpfio.netlify.app/charter)
2. Confirm that everything looks and works as expected

### CODE REVIEW

File: src/pages/charter.js
Comment (qmonnet) [inline src/pages/charter.js:249]: This link seems broken in the preview, it misses the `https://` prefix.

File: src/pages/charter.js
Comment (qmonnet) [inline src/pages/charter.js:253]:
```suggestion
        <li>All Members must abide by The Linux Foundation’s Antitrust Policy available at <a href="https://www.linuxfoundation.org/antitrust-policy" target="_blank" rel="noopener noreferrer">https://www.linuxfoundation.org/antitrust-policy</a>.</li>
```

File: src/pages/charter.js
Comment (qmonnet) [inline src/pages/charter.js:6]: This should be updated to reflect the current page content?

File: src/pages/charter.js
Comment (qmonnet) [inline src/pages/charter.js:15]: These should also be updated, `contribute` and `getting started` in particular don't sound relevant for this page.

File: src/pages/charter.js
Comment (qmonnet) [inline src/pages/charter.js:17]: This field doesn't look right, please double-check. Can you also make sure that all relevant metadata fields are updated for the new page?

**Review by qmonnet — CHANGES_REQUESTED** [review]

File: src/pages/charter.js
Comment (vannyle) [reply src/pages/charter.js]: Thanks for pointing that out. I will update the links.

File: src/pages/charter.js
Comment (vannyle) [reply src/pages/charter.js]: The keywords are updated, replacing  `contribute` and `getting started` with `charter`.

**Review by dthaler — APPROVED** [review]

File: src/pages/charter.js
Comment (dthaler) [inline src/pages/charter.js:285]:
```suggestion
      <h2>12) General Rules and Operations.</h2>
         <span>The Directed Fund activities must:</span>
```

**Comment by tgraf** [comment]
LGTM. Let's wait for final review. It matches the correct version of the charter now.

**Comment by dthaler** [comment]
@borkmann any objection to merging?

**Pull request merged**
